gef.py

```diff
@@ -2056,7 +2068,7 @@ class Architecture(metaclass=abc.ABCMeta):
     @abc.abstractproperty
     def all_registers(self):                       pass
     @abc.abstractproperty
-    def instruction_length(self):                  pass
+    def instruction_length(self) -> Optional[int]:                  pass
```

Comment:
nit: If we are keeping these as one liners, then align the `pass`, OR put the pass right after the colon.. It's fine to break them into multiple lines, though,